ProjectSettings/TagManager.asset

```diff
@@ -6,6 +6,7 @@ TagManager:
   tags:
   - PxVob_zCVobSpot
   - Climbable
+  - Subtitle
```

Comment:
Why do you add a tag? Just checking for justification. The other two times were for performance of lookup (PxVob_zCVobSpot) and climbing feature check (Climbable).

Reply:
Due to @JucanAndreiDaniel suggestion on how to centralize font swapping. Every tmp text with the tag subtitle receives the subtitle font.